### DISCUSSION TITLE
I had an error connecting apexcharts net::ERR_ABORTED 504 (Gateway Timeout). Як це виправити

### DISCUSSION THREAD

**Discussion by SvyatStupak** [apexcharts/apexcharts.js#3532] [Q&A]
I have a project on Vue 3
I installed plugin:
  npm install --save apexcharts
  npm install --save vue3-apexcharts
Then 
<img width="413" alt="Знімок екрана 2022-12-02 о 15 49 16" src="https://user-images.githubusercontent.com/48685109/205308054-e2a7fa35-6f65-4ec0-8701-c0242e21e82d.png">
Add to component
<img width="597" alt="Знімок екрана 2022-12-02 о 15 53 12" src="https://user-images.githubusercontent.com/48685109/205308263-70a9daa0-d8f9-4bbf-a2ae-4ade209e64ab.png">
and got an error in the console

app.js:8  GET http://[::1]:5173/node_modules/.vite/deps/vue3-apexcharts.js?v=9f0bad16 net::ERR_ABORTED 504 (Gateway Timeout)

**Comment by AurelianSpodarec** [comment]
So what is the solution to this?

**Comment by SousaWellington** [comment]
I  had this error.
The solution was:  `npm cache clean --force` 

and reinstall again with: 
`npm install --save apexcharts`
`npm install --save vue3-apexcharts`
